tests/ops/models/test_manual_webhook.py

```diff
@@ -0,0 +1,41 @@
+from fidesops.ops.models.manual_webhook import AccessManualWebhook
+
+
+class TestManualWebhook:
+    def test_manual_webhook(
+        self, integration_manual_webhook_config, access_manual_webhook
+    ):
+        assert (
+            integration_manual_webhook_config.access_manual_webhook
+            == access_manual_webhook
+        )
+        assert (
+            access_manual_webhook.connection_config == integration_manual_webhook_config
+        )
+
+    def test_get_enabled_no_enabled_webhooks(self, db):
+        assert AccessManualWebhook.get_enabled(db) == []
+
+    def test_get_enabled_webhooks(self, db, access_manual_webhook):
+        assert AccessManualWebhook.get_enabled(db) == [access_manual_webhook]
+
+    def test_get_enabled_webhooks_connection_config_disabled(
+        self, db, access_manual_webhook, integration_manual_webhook_config
+    ):
```

Comment:
`access_manual_webhook` doesn't get used

```suggestion
    def test_get_enabled_webhooks_connection_config_disabled(
        self, db, integration_manual_webhook_config
    ):
```

Reply:
it does get used, it needs to exist here! I need to remember the fixture you showed me before to denote this.

Reply:
ah yes, @pytest.mark.usefixtures("access_manual_webhook")